views/multiscore.erb

```diff
@@ -0,0 +1,14 @@
+<h1>Get a score for multiple words!</h1>
+<p>
+  Enter a list of comma separated words below.
+</p>
+<form id="get-multi-scores" action="#" method="post">
+  <input type="text" id="words" name="words">
+  <input type="submit" name="submit" id="submit" value="Score your words">
+</form>
+<p>
```

Comment:
Might be good to wrap this line and the rest of the page in a Ruby if statement like `<% if @words_hash %>`